server/README.md

```diff
@@ -164,6 +164,14 @@ To set the welcome theme, use the following environment value :
 To add a custom provider extend the Keycloak image and add the provider to the `/opt/jboss/keycloak/standalone/deployments/`
 directory.
 
+## Running custom scripts on startup
```

Comment:
Add to documentation that scripts can be added by extending image or by mounting.

Reply:
I believe this hasn't been properly resolved yet. 

I would like to request some additional examples that demonstrate how to mount such scripts into the image (with `docker -v` option).

Reply:
This better?

Reply:
Yes, thank you!